scp/tickets.php

```diff
@@ -129,6 +129,12 @@
                 //If no errors - them attempt the transfer.
                 if(!$errors && $ticket->transfer($_POST['deptId'], $_POST['transfer_comments'])) {
                     $msg = sprintf(__('Ticket transferred successfully to %s'),$ticket->getDeptName());
+
+                    //unassign ticket
+                    if($dept->unassignTicketsDept()=="on") {
```

Comment:
This should be a boolean function:
```php
if ($dept->unassignOnTransfer()) {
    $ticket->unassign();
}
```
Can we get rid of "on" and just use 1 or 0?